### PR TITLE
Report on maintainers' meeting

### PR DESCRIPTION
**Pull request by gvwilson**
For @k8hertweck to review.

### CODE REVIEW

File: _posts/2016/10/2016-10-13-maintainers-meeting.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+authors: ["Kate Hertweck"]
```
Comment (k8hertweck) [inline _posts/2016/10/2016-10-13-maintainers-meeting.md:3]: Add yourself as author since you put it together? :)

**Comment by gvwilson** [comment]
Naw, I just did a bit of editing.

File: _posts/2016/10/2016-10-13-maintainers-meeting.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+---
+
+This week's meeting of our lesson maintainers made some great progress in streamlining our decision-making process,
+and we hope to begin implementing some of the changes discussed in the next few weeks.
+The major highlights are:
+
+1. **Creation of a developers' subcommittee**: We're creating a new
+   subcommittee that will be in charge of decision-making for template
+   and style changes common to all lessons. Each maintainer will still
+   be responsible for PRs/issues specific to their lesson. I will
```
Comment (k8hertweck) [inline _posts/2016/10/2016-10-13-maintainers-meeting.md:17]: Add "If anyone not currently operating as a maintainer would like to assist in this new subcommittee, please [contact Kate Hertweck](mailto:k8hertweck@gmail.com)."

**Comment by k8hertweck** [comment]
I'm fine with it as is. Thanks so much for helping me get to this in a timely manner.

**Comment by gvwilson** [comment]
Ready to merge.

**Pull request merged**